### PR TITLE
Main exhibit

### PR DESCRIPTION
**Pull request by haleykim227**
Added texture image for obelisk and the obelisk with the gold pyramid tip itself.

### CODE REVIEW

**Comment by asywe16** [comment]
looking good, could you just delete the two .DS_Store files from the commit? tyty :D

File: wonderland.js [hunk 1]
```diff
@@ -2,16 +2,20 @@ import { tiny, defs } from "./external/common.js";
 import { Shape_From_File } from "./external/obj-file-demo.js";
 import { Custom_Movement_Controls } from "./custom-movement.js";
 
-const { Cube } = defs;
+const { Cube, Torus } = defs;
+const {
+    Vector, Vector3, vec, vec3, vec4, color, hex_color, Shader, Matrix, Mat4, Light, Shape, Material, Scene, Texture,
```
Comment (Rohanator9000) [inline wonderland.js]: where are you using `hex_color`?

File: wonderland.js [hunk 2]
```diff
@@ -2,16 +2,20 @@ import { tiny, defs } from "./external/common.js";
...
 
-const { Scene, Mat4, vec4, Light, color, Material, Texture } = tiny;
 export class Wonderland extends Scene {
     constructor() {
         super();
 
         this.shapes = {
             cube: new Cube(),
             walls: new Shape_From_File("assets/walls.obj"),
+            obelisk_base: new Cube(),
```
Comment (Rohanator9000) [inline wonderland.js:15]: why not just use `shapes.cube`?

File: wonderland.js [hunk 3]
```diff
@@ -27,7 +31,28 @@ export class Wonderland extends Scene {
...
+
+        // STUPID WAY OF DOING THIS
+        this.shapes.obelisk_base.arrays.texture_coord = [
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 1), Vector.of(1, 1), // bottom face
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 1), Vector.of(1, 1), // top face
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+        ]
```
Comment (Rohanator9000) [inline wonderland.js:53]: Ya what's the alternative to this? You're trying to prevent the image from stretching, right?

File: wonderland.js (hunk 2 above)
Comment (Rohanator9000) [reply wonderland.js:15]: I realize it's because you change `shapes.obelisk_base.arrays.texture_coord`.

File: wonderland.js [hunk 4]
```diff
@@ -2,16 +2,20 @@ import { tiny, defs } from "./external/common.js";
 import { Shape_From_File } from "./external/obj-file-demo.js";
 import { Custom_Movement_Controls } from "./custom-movement.js";
 
-const { Cube } = defs;
+const { Cube, Torus } = defs;
```
Comment (Rohanator9000) [inline wonderland.js]: Are you using `Torus` for anything?

File: wonderland.js (hunk 3 above)
Comment (haleykim227) [reply wonderland.js:53]: Oh there's just a quick way of doing it with one line if it's the same for all 6 faces, but this is actually the only way for doing this, bc i need 2 of the faces to have different texture than the other 4.

File: wonderland.js (hunk 4 above)
Comment (haleykim227) [reply wonderland.js]: I did, and removed it and forgot to remove.

File: wonderland.js (hunk 1 above)
Comment (haleykim227) [reply wonderland.js]: I thought I'd add all the ones from prev projects bc we might need it soon, but I can delete the ones that haven't been used yet if you'd like.

File: wonderland.js (hunk 3 above)
Comment (haleykim227) [reply wonderland.js:53]: Just removed unused imports!

**Pull request merged**